client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -3,25 +3,36 @@ import Box from '@material-ui/core/Box';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
 import { Typography } from '@material-ui/core';
+import LogoHeader from '../LogoHeader/LogoHeader';
 
 interface Props {
-  linkTo: string;
-  asideText: string;
   btnText: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ btnText }: Props): JSX.Element => {
   const classes = useStyles();
+  const becomeSitterLink = '/become-sitter';
+  const loginLink = '/login';
+  const signupLink = '/signup';
 
   return (
-    <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
-      <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
-          {btnText}
-        </Button>
-      </Link>
-    </Box>
+    <LogoHeader>
+      <Box p={1} className={classes.authHeader}>
+        <Link to={becomeSitterLink} className={classes.link}>
+          <Typography className={classes.accAside}>Become a sitter</Typography>
```

Comment:
```suggestion
          <Typography component={Link} to="/become-sitter" className={classes.accAside}>Become a sitter</Typography>
```
The react router Link and MUI's Typography can be combined :)

Reply:
Thanks for your suggestion.